### PR TITLE
fix std::move warnings from gcc

### PR DESCRIPTION
**Pull request by cyyever**
cc @EikanWang @jgong5

### CODE REVIEW

File: torch/csrc/jit/mobile/parse_bytecode.cpp
Comment (cyyever) [reply torch/csrc/jit/mobile/parse_bytecode.cpp:27]: I want to remove other std::move also, but due to incomplete support of C++17 on gcc-7, they may fail

Comment (Skylion007) [reply torch/csrc/jit/mobile/parse_bytecode.cpp:27]: We have a PR to drop GCC-7 support for this reason.

Comment (cyyever) [reply torch/csrc/jit/mobile/parse_bytecode.cpp:27]: I see, may be we can wait until it is merged.

Comment (cyyever) [reply torch/csrc/jit/mobile/parse_bytecode.cpp:27]: See commit d02959e

Comment (cyyever) [reply torch/csrc/jit/mobile/parse_bytecode.cpp:27]: We can't just return the variable because its type is not the return type and clang says 'local variable 'var' will be copied despite being returned by name'

**Comment by cyyever** [comment]
@Skylion007 Now we have no g++7, it should work

**Comment by cyyever** [comment]
cc @Skylion007 I changed the PR focus on problematic code locations and it can be merged first.

**Review by Skylion007 — APPROVED** [review]

**Pull request closed without merging**
